kratos/solving_strategies/schemes/scheme.h

```diff
@@ -746,36 +746,36 @@ class Scheme
         LocalSystemMatrixType& LHS_Contribution,
         LocalSystemVectorType& RHS_Contribution,
         Element::EquationIdVectorType& EquationId,
-        ProcessInfo& CurrentProcessInfo)
+        ProcessInfo& current_process_info)
     {
         pCurrentCondition->CalculateLocalSystem(
-            LHS_Contribution, RHS_Contribution, CurrentProcessInfo);
+            LHS_Contribution, RHS_Contribution, current_process_info);
     }
 
     virtual void Condition_Calculate_RHS_Contribution(
         Condition::Pointer pCurrentCondition,
         LocalSystemVectorType& RHS_Contribution,
         Element::EquationIdVectorType& EquationId,
-        ProcessInfo& CurrentProcessInfo)
+        ProcessInfo& current_process_info)
     {
-        pCurrentCondition->CalculateRightHandSide(RHS_Contribution, CurrentProcessInfo);
+        pCurrentCondition->CalculateRightHandSide(RHS_Contribution, current_process_info);
     }
 
     virtual void Condition_Calculate_LHS_Contribution(
         Condition::Pointer pCurrentCondition,
         LocalSystemMatrixType& LHS_Contribution,
         Element::EquationIdVectorType& EquationId,
-        ProcessInfo& CurrentProcessInfo)
+        ProcessInfo& current_process_info)
     {
-        pCurrentCondition->CalculateLeftHandSide(LHS_Contribution, CurrentProcessInfo);
+        pCurrentCondition->CalculateLeftHandSide(LHS_Contribution, current_process_info);
     }
 
     virtual void Condition_EquationId(
         Condition::Pointer pCurrentCondition,
         Element::EquationIdVectorType& EquationId,
-        ProcessInfo& CurrentProcessInfo)
```

Comment:
and definitely i would not change the signature of the functions

Reply:
Wrong correction, corrected